languages/rust/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -1,8 +1,15 @@
-There are two different types of numbers in Rust:
+## numbers
+
+There are two different types of numbers in Rust.
```

Comment:
It's a little confusing to say that there are two different types, when strictly speaking, there are 14 numeric types in Rust IIRC. 

```suggestion
There are two different categories of numbers in Rust.
```